examples/demo/scikit_estimator.py

```diff
@@ -86,16 +87,19 @@ def __call__(self, loader):
 # Instantiate an active learning strategy
 al_strategy = LeastConfidenceStrategy()
 
+# Instantiate an oracle (in this case a dummy one)
+oracle = DummyOracle()
+
 # Given that we have a data manager, a model, and an active learning strategy
 # we may create an active learning pipeline
-pipeline = GenericPipeline(data_manager=data_manager, model=model, strategy=al_strategy)
+pipeline = GenericPipeline(data_manager=data_manager, model=model, strategy=al_strategy, oracle=oracle)
```

Comment:
should we have DummyOracle by default in GenericPipeline?

Reply:
also should we rename to BenchmarkOracle? DefaultOracle?

Reply:
I think BenchmarkOracle is most indicative about its function. Will change it to that.

Reply:
Done